core/src/main/java/io/th0rgal/oraxen/utils/inventories/ItemsView.java

```diff
@@ -129,30 +129,41 @@ private record GuiItemSlot(ItemStack itemStack, Integer slot) {
     private GuiItemSlot getGuiItemSlot(final File file) {
         ItemStack itemStack;
         String fileName = Utils.removeExtension(file.getName());
-        String icon = settings.getString(String.format("oraxen_inventory.menu_layout.%s.icon", fileName), "PAPER");
+        //Material of category itemstack. if no material is set, set it to the first item of the category
+        Optional<String> icon = Optional.ofNullable(settings.getString(String.format("oraxen_inventory.menu_layout.%s.icon", fileName)));
         String displayName = ItemParser.parseComponentItemName(settings.getString(String.format("oraxen_inventory.menu_layout.%s.displayname", fileName), "<green>" + file.getName()));
-        try {
-            itemStack = new ItemBuilder(OraxenItems.getItemById(icon).getReferenceClone())
-                    .addItemFlags(ItemFlag.HIDE_ATTRIBUTES)
-                    .setDisplayName(displayName)
-                    .setLore(new ArrayList<>())
-                    .build();
-        } catch (final Exception e) {
-            try {
-                itemStack = new ItemBuilder(Material.getMaterial(icon.toUpperCase()))
+
+        itemStack = icon
```

Comment:
This can be changed to either use the provided icon or if there is no valid itemid defined as a slot, just use the first in the category, no need for material logic here